spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe 'GET /show' do
+  it 'returns a successful response' do
+    user = User.create!(name: 'Timmy')
+    get "/users/#{user.id}/posts/index"
+    puts response.status
+    expect(response).to be_successful
+  end
+
+  it 'renders a correct template' do
+    user = User.create!(name: 'Timmy')
+    get "/users/#{user.id}/posts"
+    puts response.status
+    expect(response.body).to match(/This is a list of posts for a given user/)
+  end
+end
```

Comment:
- [ ] Also here great job writing tests for your Users Post. However kindly note that you have two view files per folder, for the Users Post Index and the `Users Post Show`. for each file, you should write tests to ensure the `correct template was rendered`, `the response body includes correct placeholder text.`, and to check `If response status was correct`.

- Kindly add a render test for the User Post Show page
- Also add three missing tests for the User Post Index